assets/scss/modules/_box.scss

```diff
@@ -0,0 +1,10 @@
+@mixin dp-box ($color, $width){
+  border-radius: $dp-border-radius;
+  width: $width;
+  max-width: 408px;
+  padding: 40px 30px;
```

Comment:
Are we sure this boxes will have the same padding in handheld devices? If not, please use ems.

Reply:
ok i change it to ems